server/src/main/java/org/opentosca/toscana/plugins/cloudfoundry/CloudFoundryFileCreator.java

```diff
@@ -81,13 +97,16 @@ private void createEnvironmentVariables() throws IOException {
      add it to the manifest
      */
     private void createService() throws IOException {
-        ArrayList<String> services = new ArrayList<>();
-        services.add(String.format("  %s:", SERVICE.getName()));
-        for (Map.Entry<String, CloudFoundryServiceType> service : app.getServices().entrySet()) {
-            services.add(String.format("    - %s", service.getKey()));
-        }
-        for (String service : services) {
-            fileAccess.access(MANIFEST).appendln(service).close();
+        Map<String, CloudFoundryServiceType> appServices = app.getServices();
+        if (!appServices.isEmpty()) {
```

Comment:
maybe a nullcheck is needed here? if app.getServices() is null, nullerror will be thrown

Reply:
I dont think so, because the app.getServices() cant be null. It returns always a list as you can see in the CloudFoundryApplication class. 
I think it is sufficient to check if the list is empty